include/seqan/modifier/modifier_iterator.h

```diff
@@ -538,11 +530,20 @@ position(ModifiedIterator<THost, TSpec> const & me)
     return position(host(me));
 }
 
+// rooted overload
+template <typename TContainer1, typename TIterator, typename TSpec1, typename TSpec2, typename TContainer2>
+inline typename Position<ModifiedIterator<Iter<TContainer1, AdaptorIterator<TIterator, TSpec1> >, TSpec2> const>::Type
+position(ModifiedIterator<Iter<TContainer1, AdaptorIterator<TIterator, TSpec1> >, TSpec2> const & me,
+        TContainer2 const &)
```

Comment:
Is there no global rooted iterator interface that could be called instead of overloading it?